javatests/arcs/showcase/typevariables/OrderIngestion.kt

```diff
@@ -0,0 +1,39 @@
+package arcs.showcase.typevariables
+
+import kotlinx.coroutines.Job
+
+class OrderIngestion : AbstractOrderIngestion() {
+
+    /** A user places an order of three items */
+    override fun onReady() {
+        if (!orderedOnce.isCompleted) {
```

Comment:
You don't really need this clause, your particle will not be started multiple times in the test.